usaspending_api/settings.py

```diff
@@ -277,7 +277,7 @@
         'console': {
             'handlers': ['console', 'console_file'],
             'level': 'INFO',
-            'propagate': True,
+            'propagate': False,
```

Comment:
Thanks @willkjackson for the recommendation to change this to stop double logging